examples/styled-components/README.md

```diff
@@ -0,0 +1,2229 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Let's describe example here
how to build it
and what it uses inside

Reply:
Instruction in the readme is correct. `npm i && npm start`

Reply:
I mean it huge Readme
there is no need of it
we need to describe what is special in these particular example